src/templates/article.liquid

```diff
@@ -16,7 +16,11 @@
 <article role="article" itemscope itemtype="http://schema.org/Article">
 
   {% if article.image %}
-    {{ article | img_url: '1024x1024' | img_tag: article.title }}
+    <div class="responsive-bg-image hero-banner hero-banner--article lazyload" data-bgset="{% include 'responsive-bg-image', image: article.image %}"></div>
+
+    <noscript>
+      <div class="responsive-bg-image" style="background-image: {{ article.image | img_url: 'master' }}"></div>
```

Comment:
We decided to deprecate the use of named image sizes awhile ago. Instead of `master`, please switch this to use `image.width` to calculate what the largest image size would be.

See [Deprecated filters - Named size parameters](https://help.shopify.com/themes/liquid/filters/deprecated-filters#named-size-parameters)

Reply:
It should be good now.